### PR TITLE
Document issues tag in release metadata

### LINKED ISSUE #191: Update releases section to include `issues` tag
### Problem

We now support the `issues` tag bug our docs don't include it in the example of how to create a `release` tag

### Proposal

We should include an example of how to use the `issues` tag so developers know how to use it

### Prior Art (Optional)

https://www.freedesktop.org/software/appstream/docs/sect-Metadata-Releases.html#tag-release-issues

### PR DESCRIPTION
**Pull request by Antolius**
Resolve #191

### CODE REVIEW

File: writing-apps/our-first-app/metadata.md [hunk 1]
```diff
@@ -112,6 +112,10 @@ Your app must have a `release` tag for every version you wish to publish in AppC
         <li>An annoying behavior no longer occurs</li>
       </ul>
     </description>
+    <issues>
```
Comment (danirabbit) [inline writing-apps/our-first-app/metadata.md:115]: I think we should update the description paragraph here as well. Currently it says to "avoid technical language", but issue tags should use the same title as the linked issue report according to the spec. So perhaps we should clarify that only the `description` tag should contain a non-technical description of changes and the `issues` tag should include any and all issues resolved in this release.

File: appcenter/publishing-updates.md [hunk 2]
```diff
@@ -10,7 +10,7 @@ If you change the UI of your app in an update, you **must** update your screensh
 
 ## Release Descriptions
 
-You must add accurate release descriptions to your metainfo.xml file when publishing an update. For more information on formatting, see the [FreeDesktop.org AppStream documentation](https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html#sect-Metadata-GenericComponent) for the `<releases />` tag. The version in your release should match the latest version submitted to AppCenter.
+You must add accurate release descriptions to your metainfo.xml file when publishing an update. For more information on formatting, see the [FreeDesktop.org AppStream documentation](https://www.freedesktop.org/software/appstream/docs/sect-Metadata-Releases.html#spec-releases) for the `<releases />` tag. The version in your release should match the latest version submitted to AppCenter.
```
Comment (danirabbit) [inline appcenter/publishing-updates.md:13]: Much better link, thank you!

**Review by danirabbit — COMMENTED** [review]
This looks good nice work! Just one comment.

I kind of wonder if maybe we should combine these two sections and link from `publishing updates` to `our first app` so we're not duplicating information

File: writing-apps/our-first-app/metadata.md (hunk 1 above)
Comment (Antolius) [reply writing-apps/our-first-app/metadata.md]: How about reducing the size of this page in favor of linking to the _Publishing Updates_?

**Review by danirabbit — APPROVED** [review]
Yup, good call! This looks great. Thanks!

**Pull request merged**